Tango :: 2.2 fixes

ronso0 opened the pull request:
some small improvements to make Tango look like it did with qt4.
also, now the crossfader is visible only visible if the (channel) mixer is shown.

ronso0 committed:
Tango: remove gap in between beatgrid buttons and beatgrid buttons toggle

ronso0 committed:
Tango: remove qproperty-layoutAlignment from #SkinSettingsButtonMulti

ronso0 committed:
Tango: couple crossfader and channel mixer visibility

ronso0 committed:
Tango: round corners for loop containers

ronso0 committed:
Tango: fix position of big spinny/cover art with loop sizes expanded

ronso0 committed:
Tango: show topbar with big library, exclude irrelevant buttons then

ronso0 committed:
Tango (64 samplers): show topbar with big library, exclude irrelevant buttons then

ronso0 committed:
Tango: remove obsolete size_detector_screen.xml

ronso0 committed:
Tango: fix some tiny glitches

beenisss commented on res/skins/Tango/skin_settings.xml:
```diff
@@ -647,7 +640,7 @@ Description:
             <MinimumSize>200,26</MinimumSize>
             <MaximumSize>200,54</MaximumSize>
-            <SizePolicy>me,min</SizePolicy>
+            <SizePolicy>me,</SizePolicy>
```
This looks syntactically incorrect but I could be wrong.

beenisss commented:
Testing on Mac, these look fine to me. I can't actually see any discernible difference between the old and updated skins, not sure if that's a good thing or not.

I don't use any FX or samplers on a regular basis so would be worth double checking with somebody who uses them a lot.

beenisss commented:
The bpm library column has that same issue I mentioned on Zulip but I assume that is not relevant.

ronso0 committed:
Tango: fix missing size policy in skinsettings

ronso0 commented:
> Testing on Mac, these look fine to me. I can't actually see any discernible difference between the old and updated skins, not sure if that's a good thing or not.

expand the beatgrid buttons (in waveforms) and maximize the library, then you shoud see it.
yes, apart from that that it's just small issues and error messages I fixed.

ronso0 committed:
Tango: strip rubbish from SVGs

beenisss commented:
> expand the beatgrid buttons (in waveforms) and maximize the library, then you shoud see it.

Ah, I get you. Yup, both looking spot on.

ronso0 committed:
Tango: clean up

ronso0 commented:
Ready to merge.

ronso0 committed:
Tango: center lib radio button icons

ronso0 committed:
Tango: vecorize & unify lib icon colors

uklotzde commented:
Still LGTM. I'm planning a new snapshot build for RPM Fusion and would like to include all these nice polishing changes.
